### PR TITLE
[Don't merge] MySql: update Debezium version to 1.7.1-final

### LINKED ISSUE #6884: Source MySQL CDC: sync failed when has null value in `created_at` column
## Enviroment
Is this your first time deploying Airbyte: No
OS Version / Instance:  c5.4xlarge
Memory / Disk:  (16vCPU and 32GB ram)
Deployment: AWS Kubernetes
 Airbyte Version: 0.30.1-alpha
Step: Initial Sync
Description: We are trying to replicate from Aurora RDS MySQL Reporting Replica with Binary Logs (CDC) to AWS S3, and we have some tables that have datetime fields that either have 0 or null values.  We run into Java Heap errors like the following:

[Slack convo](https://airbytehq.slack.com/archives/C01MFR03D5W/p1633045225041800)

## Current Behavior
image

## Expected Behavior
*Tell us what should happen.*

## Logs
[logs-165-0 masked.txt](https://github.com/airbytehq/airbyte/files/7307893/logs-165-0.masked.txt)

## Steps to Reproduce
1.
2.
3.

## Are you willing to submit a PR?

Remove this with your answer.

### PR DESCRIPTION
**Pull request by DoNotPanicUA**
For the issue #6884.
This PR will not be merged, because the latest version of the driver effects Postgres source.

---
This change is [Reviewable](https://reviewable.io/reviews/airbytehq/airbyte/7932)

### CODE REVIEW

File: airbyte-integrations/connectors/source-mysql/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,6 @@ COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
 
 RUN tar xf ${APPLICATION}.tar --strip-components=1
 
-LABEL io.airbyte.version=0.4.9
+LABEL io.airbyte.version=0.4.9b
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-mysql/Dockerfile:11]: nicely done putting it as a non-main release ;)

**Comment by ChristopheDuong** [comment]
>  This PR will not be merged, because the latest version of the driver effects Postgres source.
@DoNotPanicUA How do it affect postgres source? 

There seems to be fixes on debezium that could benefit the source-postgres connector too. Have you experienced any troubles with postgres using this new debezium version?

**Comment by DoNotPanicUA** [comment]
@ChristopheDuong 
This PR was created to push subversion and check it as a fix for a customer. In case of a positive result, I was going to set the latest version and check all other sources.
As so far there is no feedback, the ticket is on hold. And to be honest I don't remember testing other sources.
But! there was one source that failed after the update. It might be a Postgress.
I would recommend you cherry-pick this change on the latest master and check it there.

**Comment by DoNotPanicUA** [comment]
#10251 fix PR

**Pull request closed without merging**